test/integration/job/job_test.go

```diff
@@ -82,6 +82,7 @@ func TestMetricsOnSuccesses(t *testing.T) {
 		job                       *batchv1.Job
 		wantJobFinishedNumMetric  metricLabelsWithValue
 		wantJobPodsFinishedMetric metricLabelsWithValue
+		wantJobPodsCreatedMetric  metricLabelsWithValue
```

Comment:
can you undo the changes here? I want to get rid of this test #121408

Reply:
done